v3/src/components/slider/slider-component.tsx

```diff
@@ -158,12 +158,7 @@ export const SliderComponent = observer(({sliderModel} : IProps) => {
               />
             </svg>
 
-            <V3SliderThumb
-              layout={layout}
-              sliderModel={sliderModel}
-              sliderVal={sliderModel.globalValue.value} //TODO - destructure both on the other side?
-              scaleDomain={sliderModel.axis.domain}
-            />
+            <CodapSliderThumb sliderModel={sliderModel} />
```

Comment:
@bacalj this is still reactive, see other side, you need to address property in dependency array, not destructure ahead of time